components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -280,6 +280,7 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
     createStripeCustomerIfNeeded(attributionId: string, currency: string): Promise<void>;
     subscribeToStripe(attributionId: string, setupIntentId: string, usageLimit: number): Promise<number | undefined>;
     getStripePortalUrl(attributionId: string): Promise<string>;
+    getCurrentBillingCycleDates(attributionId: string): Promise<{ from: string; to: string } | undefined>;
```

Comment:
nit: I'd be in favor of exposing a getCostCenter (combines `getUsageLimit`) here instead of adding a function for every property. This also would probably make it easier later to directly talk to the usage component from the dashboard through our API.

Reply:
Thanks. That's what I had initially, and it seemed to work, but it felt weird to expose a Usage proto shape (`CostCenter`) in gitpod-protocol. E.g. it was the only import with an absolute path (`@gitpod/...`) while all the other imports there are relative (`./...`).

But I'm happy to try that again if you think it's better. 👍

Reply:
duplicating the shape is fine, my comment was more about fetching the properties of a cost center in one call.